var/spack/repos/builtin/packages/meme/package.py

```diff
@@ -35,9 +35,20 @@ class Meme(AutotoolsPackage):
 
     version('4.11.4', '371f513f82fa0888205748e333003897')
 
+    variant('mpi', default=False, description='Enable MPI support')
+
     depends_on('zlib', type=('link'))
     depends_on('libxml2', type=('link'))
     depends_on('libxslt', type=('link'))
     depends_on('libgcrypt', type=('link'))
     depends_on('perl', type=('build', 'run'))
     depends_on('python@2.7:', type=('build', 'run'))
+    # openmpi support
+    depends_on('openmpi@2:', when='+mpi', type=('build', 'run'))
```

Comment:
Bad habits die hard.

I see that when I change to link then recompile

"readelf -d package | grep 'R.*PATH'" shows openmpi.  So that's the correct one.  I'll remember this going forward.